source/server/connection_handler_impl.cc

```diff
@@ -125,10 +137,13 @@ ConnectionHandlerImpl::ActiveTcpListener::~ActiveTcpListener() {
     parent_.dispatcher_.deferredDelete(std::move(removed));
   }
 
-  while (!connections_.empty()) {
-    connections_.front()->connection_->close(Network::ConnectionCloseType::NoFlush);
+  for (auto& tagAndConnections : connections_by_tag_) {
```

Comment:
Nit: `tag_and_connections`

Reply:
err, context switching between two styles is hard. fixed